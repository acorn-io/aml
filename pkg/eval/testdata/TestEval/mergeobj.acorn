a: arr: [1,{x:2}]
a: arr: [1,{x:2}]
a: b: 1
a: c: 2
a: b: 1
