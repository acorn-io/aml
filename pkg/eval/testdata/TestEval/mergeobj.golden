{
  "a": {
    "arr": [
      1,
      {
        "x": 2
      }
    ],
    "b": 1,
    "c": 2
  }
}